SECURITY.md

```diff
@@ -0,0 +1,6 @@
+# Security Policies and Procedures
+
+Security reports should never start out in the open. Please follow up directly
+with the team if you have a contact. If not you can always start with the
+information at https://docs.chocolatey.org/en-us/information/security to see

```

Comment:
Maybe? That is a pretty long document, and the contact email is all the way at the bottom.
```suggestion
information at https://docs.chocolatey.org/en-us/information/security#report-issue to see

```